react-chat/src/App.jsx

```diff
@@ -1,99 +1,91 @@
-import React, { useState, useEffect } from 'react';
-import { Routes, Route, Navigate, useParams } from 'react-router-dom';
+import React, { useState, useEffect, useCallback } from 'react';
+import { Routes, Route, Navigate, useParams, useNavigate } from 'react-router-dom';
 import ChatList from './components/ChatList/ChatList';
 import Chat from './components/Chat/Chat';
 import Profile from './components/Profile/Profile';
+import Login from './components/Login/Login';
+import Register from './components/Register/Register';
 import AppContext from './context/AppContext';
-import { avatars, defaultAvatar } from './assets/avatars';
+import Api from './api/api';
```

Comment:
Если файл будет называться `index.js`, то можно будет импортировать из `api`, а не из `api/api`.

```suggestion
import { Api } from './api';
```